src/types.ts

```diff
@@ -2241,6 +2241,11 @@ export type Campaign = {
 } & CampaignData &
   CampaignStatus;
 
+export type TestCampaignResponse = {
+  campaign?: Campaign;
```

Comment:
in which situation would this field be empty?

Reply:
when any user is invalid, the test is not ran